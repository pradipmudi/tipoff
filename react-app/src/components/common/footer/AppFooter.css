#footer {
    background-color: #e1f4f3;
}