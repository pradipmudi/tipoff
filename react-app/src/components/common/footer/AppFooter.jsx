import React from 'react';

import { 
    Container, 
    Icon,
    Content,
    Columns,
    Column,
    Footer
 } from 'bloomer';

 import './AppFooter.css';

const AppFooter = () => {
    return (<Footer id='footer'>
                <Container>
                    <Content>
                        <Columns>
                            <Column hasTextColor="black">
                                <p>
                                    Made with<Icon hasTextColor="danger" className="fas fa-heart"></Icon> 
                                    by <a href="https://github.com/tanha-p" target="_new">
                                    <Icon className="fab fa-github"></Icon> tanha-p</a> and these <a href="https://stackshare.io/tipoff/tipoff" target="_new">awesome techs</a>
                                </p>
                            </Column>
                        </Columns>
                    </Content>
                </Container>
            </Footer>);
}

export default AppFooter;