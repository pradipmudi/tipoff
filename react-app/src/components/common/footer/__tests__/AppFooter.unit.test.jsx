import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';

import AppFooter from '../AppFooter';


describe("AppFooter", () => {
    it("matches snapshot", () => {
        const snap = renderer.create(<AppFooter />).toJSON();
        expect(snap).toMatchSnapshot();
    });

    it("renders without issues", () => {
        const wrapper = shallow(<AppFooter />);
        expect(wrapper.exists()).toEqual(true);
    });
});