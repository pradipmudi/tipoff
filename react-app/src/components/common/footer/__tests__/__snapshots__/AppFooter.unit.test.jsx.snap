// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter matches snapshot 1`] = `
<footer
  className="footer"
  id="footer"
>
  <div
    className="container"
  >
    <div
      className="content"
    >
      <div
        className="columns"
      >
        <div
          className="column has-text-black"
        >
          <p>
            Made with
            <span
              className="icon"
            >
              <i
                aria-hidden="true"
                className="has-text-danger fas fa-heart"
              />
            </span>
            by 
            <a
              href="https://github.com/tanha-p"
              target="_new"
            >
              <span
                className="icon"
              >
                <i
                  aria-hidden="true"
                  className="fab fa-github"
                />
              </span>
               tanha-p
            </a>
             and these 
            <a
              href="https://stackshare.io/tipoff/tipoff"
              target="_new"
            >
              awesome techs
            </a>
          </p>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
